projects/plugins/jetpack/extensions/plugins/post-publish-qr-post-panel/components/qr-post.js

```diff
@@ -63,3 +65,41 @@ export default function QRPost() {
 		</div>
 	);
 }
+
+export function QRPostButton() {
+	const qrCodeRef = useRef();
+	const slug = useSelect( select => select( editorStore ).getEditedPostSlug(), [] );
+	const [ isModalOpen, setIsModalOpen ] = useState( false );
+	const switchModal = () => setIsModalOpen( v => ! v );
+	const closeModal = () => setIsModalOpen( false );
+
+	return (
+		<div className="qr-post-button">
```

Comment:
Here and on line 92 (and 88) - I notice the class names aren't prefixed with anything like `jetpack-` or `jp-`. Mainly mentioning it because of the possibility of conflicts with other plugins. It's the class name on line 77 that seems most likely to result in a conflict I think in the future - but it seems pretty safe just now checking at https://wpdirectory.net/.
I'm not sure if you want to take or leave this but wanted to add it anyway just in case :)

Reply:
That's a good call. It may be a good idea to be more specific, in a follow-up PR.

Reply:
https://github.com/Automattic/jetpack/issues/23428